xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1500,18 +1609,23 @@ The current depth exceeds the recursive limit set by the maximum depth.</excepti
         <ReturnType>System.Buffers.ReadOnlySequence&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the value of the last processed token as a <see cref="T:System.ReadOnlySpan`1" /> slice of the input payload. If the JSON is provided within a <see cref="T:System.Buffers.ReadOnlySequence`1" /> and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
+        <summary>Gets the raw value of the last processed token as a ReadOnlySequence&lt;byte&gt; slice of the input payload. If the JSON is provided within a ReadOnlySequence&lt;byte&gt; and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
         <value>A byte read-only sequence.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
 
-If <xref:System.Text.Json.Utf8JsonReader.HasValueSequence> is `false`, `ValueSequence` is empty. 
+If <xref:System.Text.Json.Utf8JsonReader.HasValueSequence> is `false`, <xref:System.Text.Json.Utf8JsonReader.ValueSequence> is empty. 
 Therefore, only access `ValueSequence` if <xref:System.Text.Json.Utf8JsonReader.HasValueSequence> is `true`.
 Otherwise, the token value must be accessed from <xref:System.Text.Json.Utf8JsonReader.ValueSpan>.
 
+There is no guarantee that this property will always contain well-formed data. If the input JSON passed in to the Utf8JsonReader contains invalid UTF-8 bytes within JSON string tokens, this property will return back those invalid UTF-8 bytes as is.
+Therefore, if the input is untrusted or not previously validated, call <xref:System.Text.Json.Utf8JsonReader.GetString> to get the JSON string token.
+
+Since this property returns the raw bytes, avoid using it for text comparison. Instead call <xref:System.Text.Json.Utf8JsonReader.ValueTextEquals> which will unescape the text if necessary.
```

Comment:
```suggestion
Since this property returns the raw bytes, avoid using it for text comparison. Instead call <xref:System.Text.Json.Utf8JsonReader.ValueTextEquals>, which will unescape the text if necessary.
```

Reply:
How could I see this and know to use %2A to fix it? I had seen something similar for generics (%601). How do I interpret these custom escaped characters?

Reply:
@ahsonkhan, those are URL encoded characters, %2A ("*") is used to refer to any signature of the `ValueTextEquals` method.
%60 ("\`") is the tilde which is used to refer to the generic-type members i.e: \<xref:System.Span`1\>; this tag is equivalent to \<xref:System.Span%601\>.
I have seen both ways used within the docs, with no actual difference between each other when they render.